### ISSUE TITLE
Couldn't find the UE4/UE5 workspace

### ISSUE THREAD

**Issue by Anti-Distinctlyminty** [boocs/ue4-intellisense-fixes#4]
The extension doesn't work for me. I'm just getting the following error:

```
Couldn't find the UE4/UE5 workspace.
** Error **: Couldn't parse Unreal Engine version.
** Error **: No fixable project could be created.
```

I'm using UE 4.27

**Comment by boocs** [comment]
@Anti-Distinctlyminty 

This should've been fixed with the latest extension version which had special fixes for 4.27.0. Make sure you're running extension version 2.1.0 and not 2.0.1.

**Comment by boocs** [comment]
``` Couldn't find the UE4/UE5 workspace.```
This error is very strange and should never happen unless something is wrong.

1. Did you open your project as a folder or a workspace? 
On your VSCode title bar does it contain (Workspace) in the title?

   You need to double click the *.code-workspace file, in your project directory, to open the project as a workspace.

2. If you done the above and it still errors. Open the *.code-workspace file and tell me what it says under folders:
  ```
  "folders": [
		{
			"name": "MyProject",
			"path": "."
		},
		{
			"name": "UE4",
			"path": "K:\\Program Files\\Epic Games\\UE_4.27"
		}
	]
  ```
  This is how I get the UE4/UE5 path in my extension. What does it say for the name of UE4?

3.  If you don't have a *.code-workspace file in your project directory then you need to go into the Unreal Engine and change it to work with VSCode. 

  Go to Editor preferences->Source Code and change the dropdown menu to Visual Studio Code. After the Unreal Engine 
  restarts you might have the file or might not. If you still don't, go to the Unreal Engine File menu and choose Refresh Visual 
  Studio Code Project.
